test/unit/get-config-test.js

```diff
@@ -970,4 +970,36 @@ describe('getRuleFromString', function () {
       );
     }
   });
+
+  it('supports a `.template-lintrc.mjs` config file', async function () {
+    let project = await Project.defaultSetup();
+
+    project.write({
+      '.template-lintrc.mjs': `module.exports = { extends: 'recommended' };`,
```

Comment:
should we use ESM code `export default = ...` in this example given that it's mjs?

Reply:
Yup, oversight on my part. Updated.